src/cpu/platform.cpp

```diff
@@ -119,6 +119,8 @@ bool has_data_type_support(data_type_t data_type) {
         case data_type::f16:
 #if DNNL_X64
             return x64::mayiuse(x64::avx512_core_fp16);
+#elif DNNL_AARCH64_USE_ACL
```

Comment:
It's worth noting that fp16 support in ACL will be dependent on how the ACL lib was built.

Reply:
Yes, as you said, we need to enable multi_isa options.